app/controllers/website_members_controller.rb

```diff
@@ -10,8 +11,11 @@ def update
   end
 
   def change_role
-    @website_member.update_attributes(website_member_role_params) if website_member_role_params[:role].present?
-    redirect_to settings_url, notice: 'Role updated'
+    current_website_member = WebsiteMember.find_by_user_id_and_website_id(current_user.id, current_website.id)
+    if current_website_member.role.owner?
+      @website_member.update_attributes(website_member_role_params)
+      render json: @website_member
+    end
```

Comment:
I want you to confirm that if a user tries to change the role we are rendering a json and telling them that they don't have permission. Is that correct?